fv3core/stencils/updatedzc.py

```diff
@@ -126,24 +124,7 @@ def __call__(
             ws: surface vertical wind implied by horizontal motion over topography
             dt: timestep over which to evolve the geopotential height
         """
-        basic_operations.copy_stencil(
-            gz,
-            self._gz_in,
-            origin=self.grid.full_origin(),
-            domain=self.grid.domain_shape_full(add=(0, 0, 1)),
-        )
-        basic_operations.copy_stencil(
-            gz,
-            self._gz_x,
-            origin=self.grid.full_origin(),
-            domain=self.grid.domain_shape_full(add=(0, 0, 1)),
-        )
-        basic_operations.copy_stencil(
-            gz,
-            self._gz_y,
-            origin=self.grid.full_origin(),
-            domain=self.grid.domain_shape_full(add=(0, 0, 1)),
-        )
+        self._double_copy_stencil(gz, self._gz_x, self._gz_y)
```

Comment:
Should add a TODO with the plans you currently have for this object, since that information is easy to lose over time. Feel free to use this or write your own comment.

```suggestion
        # TODO: delete gz_x and gz_y and just use gz when regions bug is fixed
        # currently doing so breaks validation
        self._double_copy_stencil(gz, self._gz_x, self._gz_y)
```